README.md

```diff
@@ -1,79 +1,22 @@
-[FSI tutorial](https://github.com/precice/precice/wiki/FSI-tutorial)
-- of-of coupled simulation with OpenFOAM
-[Tutorial for CHT: Flow over a heated plate](https://github.com/precice/openfoam-adapter/wiki/Tutorial-for-CHT:-Flow-over-a-heated-plate)
-- of-ccx conjugate heat transfer simulation with OpenFOAM and CalculiX
-[Tutorial for CHT with OpenFOAM and CalculiX](https://github.com/precice/precice/wiki/Tutorial-for-CHT-with-OpenFOAM-and-CalculiX)
-- precice Testing some components of preCICE: dummy solvers and Python bindings
-
-
-# travis.yml
-Continuous Integration config.
-Two [build stages](https://docs.travis-ci.com/user/build-stages/):
-- stage 1 (build stage): precice docker image build and pushed to [Docker Hub](https://hub.docker.com/u/precice)
-- stage 2 (testing stage): build su2-ccx, of-of, of-ccx and push outputfiles to repo
-
-# compare_*.py
-Python script to compare reference data with output data.
-
-# Others
-- Makefile_*: needed to build calculix-adapter in su2-ccx and of-ccx
-- push_*: script to push outputfiles back to rep [HowTo](https://gist.github.com/willprice/e07efd73fb7f13f917ea#file-push-sh)
-- How to push to [hub.docker.com](https://hub.docker.com/) [link1](https://docs.travis-ci.com/user/docker/#Pushing-a-Docker-Image-to-a-Registry) [link2](https://docs.travis-ci.com/user/build-stages/share-docker-image/)
-- log_*: script to log infomation about the build
-
-# How to create a new system tests
-1. Create a directory `Test_mysystemtest`.
-2. Create a `Dockerfile` in there.
-3. Create a directory `referenceOutput` there.
-4. Copy the output files to a folder `/Output/` (inside the container).
-
-## Useful paths:
-
-* preCICE Library path: `-L/home/precice/precice-install/libs` or `pkg-config --libs-only-L libprecice`
-* preCICE Include path: `-I/home/precice/precice-install/include` or `pkg-config --cflags-only-I libprecice`
-* preCICE library flag: `-lprecice` or `pkg-config --libs-only-l libprecice`
-
-# Adding CI to a new adapter
-
-As described in [#22](https://github.com/precice/systemtests/pull/22),  using `trigger_systemtests.py` it is possible to trigger custom systemtests from the travis job of the another adapter, therefore providing continous integration for the adapter on each commit to a particular branch. If you want to add another adapter, or modify tests that are run for it, you need to modify `nm_repo_map` and `nm_test_map` variables in the `trigger_systemtests.py`, that describe mapping between adapter name and repository and adapter name and set of test cases correspondingly.
+```
+   python3 local_test.py -s of-of su2-ccx -f Dockefile.Ubuntu1604.home
+``` 
 
-The adapter's Travis build would be considered successful if systemtests Travis build exits with status 'Failed' or 'Errored'. It would exit with 'Success', if the systemtests Travis build exits with 'Success' or 'Canceled'.
+More information can be found in the [docs](`www.github.com/precice/systemtests/docs`)
```

Comment:
I wanted to keep the main `README.md` minimal, otherwise no one reads it (like the current README...) . But probably overview of the what repository contains is useful. I will also add description of what scripts do.

Reply:
Keeping it short is good! But, since it's the first document one reads, it should give directions on what to expect and what to read next.